server/lib/statsd-collector.js

```diff
@@ -10,9 +10,19 @@ var uaParser = require('ua-parser');
 
 var STATSD_PREFIX = 'fxa.content.';
 
+/**
+ * Normalize the string to make it usable in StatsD tagging.
+ * @param {String} item
+ * @returns {String}
+ * @private
+ */
+function _normalizeTag(item) {
+  return String(item).replace(/ /g, '_').toLowerCase();
```

Comment:
Do we need to do any other escaping/translations/ensuring no strange characters are possible?